Description: fix FTBFS with --as-needed linker option
 Libraries must be placed after source files in command line.
Bug-Debian: http://bugs.debian.org/692307
Author: Ilya Barygin <randomaction@ubuntu.com>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/asn1c/tests/check-assembly.sh
+++ b/asn1c/tests/check-assembly.sh
@@ -48,7 +48,7 @@ CC ?= ${CC}
 all: check-executable
 check-executable: compiled-module *.c*
 	@rm -f *.core
-	\$(CC) \$(CFLAGS) -o check-executable -lm *.c*
+	\$(CC) \$(CFLAGS) -o check-executable *.c* -lm
 
 # Compile the corresponding .asn1 spec.
 compiled-module: ${asn_module} ../../asn1c
